.github/workflows/release-stable.yml

```diff
@@ -42,6 +54,7 @@ jobs:
         run: |
           git config user.name ${{ github.actor }}
           git config user.email opensource@acrolinx.com
+          git checkout -b release-$RELEASE_VERSION
```

Comment:
maybe we should not use the exact same name for the git tag and the git branch, to avoid confusion (also if something goes wrong). We could call the branch "release-stable-$RELEASE_VERSION" or something else, maybe you have better ideas?

Reply:
see if what I used works